article/app/services/dotcomponents/ArticlePicker.scala

```diff
@@ -90,9 +90,19 @@ object ArticlePageChecks {
     // See: https://github.com/guardian/dotcom-rendering/blob/master/packages/frontend/web/components/lib/ArticleRenderer.tsx
     def unsupportedElement(blockElement: BlockElement) =
       blockElement match {
-        case _: TextBlockElement  => false
-        case _: ImageBlockElement => false
-        case _                    => true
+        case _: TextBlockElement    => false
+        case _: ImageBlockElement   => false
+        case _: VideoBlockElement   => false
+        case _: GuVideoBlockElement => false
```

Comment:
Will GuVideo work?

Reply:
ah, good point, we'll need to add it in DCR, like https://github.com/guardian/dotcom-rendering/pull/2369